roles/openshift_storage_glusterfs/tasks/heketi_ops_pending.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Set Heketi server mode as read-only
```

Comment:
This has nothing to do with checking pending ops, strictly speaking. Move it out into `glusterfs_upgrade.yml`.

Reply:
done